### ISSUE TITLE
Clone VI, Class Ctrl/Ind might get renamed incorrectly

### ISSUE THREAD

**Issue by MikaelHolmstrom** [opengds/OpenGDS#18]

**Comment by kosist** [comment]
Does this issue relates to the fact, that class input/output terminals are renamed, but they don't have "in"/"out" in their names, please?

**Comment by MikaelHolmstrom** [comment]
No I think this ws that if a class VI has class referece from other classes as controls/indicators (e.g. dependency injection), those class controls also gets renamed.

**Comment by MikaelHolmstrom** [comment]
@kosist , it looks like I've fixed this issue in the latest build. Do you want a different behaviour for this function?

**Comment by kosist** [comment]
I had the following issue. Let's say, "Class A" was cloned as "Class B", and input terminals of cloned methods had names "Class B", instead of "Class B in"/"Class B out".
Right now I'm testing it with dummy classes in dummy project, but GOOP works just fine, input/output terminals are renamed properly.
If I try to clone the same class, with which I've had issue, then terminals are renamed not correctly.
Strange... I rename actor, which is inside the library. But again, if I do it with dummy project, everything works fine.
Now I've noticed, that I've got a bit out-dated GOOP toolkit (1.2.12); so I'll try to update it to 1.2.20, and try again.
